### PR TITLE
Feature/123 create lab type page

### PR DESCRIPTION
**Pull request by todd-dembrey**
Labs will submit to the core application submission but doesnt require rounds

@zerolab You might be interested in the change to the way data gets built for submission

### CODE REVIEW

File: opentech/public/funds/models.py [hunk 1]
```diff
@@ -135,6 +137,16 @@ def clean(self):
                 'lab_link': 'Please provide a way for applicants to apply',
             })
 
+        if self.lab_type and self.link_text:
+            raise ValidationError({
+                'link_text': 'You cant customise the text for internal lab pages, leave blank',
```
Comment (zerolab) [inline opentech/public/funds/models.py]: "cant" -> "cannot" or "can\'t"

**Comment by zerolab** [comment]
minor text change needed. otherwise LGTM

**Comment by todd-dembrey** [comment]
@chris-lawton Can you please review the templates for feature 71.

File: opentech/public/funds/templates/public_funds/lab_page.html [hunk 2]
```diff
@@ -3,14 +3,27 @@
 
 {% block content %}
     {% include "public_funds/includes/lab_apply_cta.html" %}
-    <div class="wrapper wrapper--flex">
+    <div class="wrapper">
         <section class="section section--main">
-            <h1>{{ page.icon }}{{ page.title }}</h1>
-            <h5>{{ page.introduction }}</h5>
+            <div class="media-box media-box--reverse">
```
Comment (zerolab) [inline opentech/public/funds/templates/public_funds/lab_page.html]: When no icon or intro are set, we'll get a funky empty set of divs. I say less markup, the better

Comment (todd-dembrey) [reply opentech/public/funds/templates/public_funds/lab_page.html]: Yeah, Chris should hopefully clear that up. It came from a stale branch so probably isnt correct.

**Pull request merged**